src/main/java/com/github/havardh/javaflow/phases/verifier/ClassGetterNamingVerifier.java

```diff
@@ -0,0 +1,70 @@
+package com.github.havardh.javaflow.phases.verifier;
+
+import static java.lang.String.format;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.github.havardh.javaflow.ast.Class;
+import com.github.havardh.javaflow.ast.Field;
+import com.github.havardh.javaflow.ast.Method;
+import com.github.havardh.javaflow.ast.Type;
+import com.github.havardh.javaflow.exceptions.AggregatedException;
+import com.github.havardh.javaflow.exceptions.FieldGettersMismatchException;
+
+public class ClassGetterNamingVerifier implements Verifier {
+
+  @Override
+  public void verify(List<Type> types) {
+    List<Exception> exceptions = new ArrayList<>();
+    for (Type type : types) {
+      if (type instanceof Class) {
+        try {
+          validate((Class) type);
+        } catch (Exception e) {
+          exceptions.add(e);
+        }
+      }
+    }
+
+    if (!exceptions.isEmpty()) {
+      throw new AggregatedException("Class getter naming validation failed", exceptions);
```

Comment:
We need to add a space after this message, either here or in the AggregatedException class.